recipes/ree.rb

```diff
@@ -1,62 +1,62 @@
 
-case node.platform_family
+case node['platform_family']
```

Comment:
I'm limiting my feedback here to saying that these are good changes. That doesn't mean there aren't further improvements that could be made, but I'm not looking for them. 😁